security/role/role.go

```diff
@@ -39,3 +39,105 @@ func (sr SecurityRole) Fetch(id string, options *types.Options) (types.Role, err
 
 	return role, nil
 }
+
+/*
+  Create a new Role in Kuzzle.
+*/
+func (sr SecurityRole) Create(id string, controllers types.Controllers, options *types.Options) (types.Role, error) {
+	if id == "" {
+		return types.Role{}, errors.New("Security.Role.Create: role id required")
+	}
+
+	action := "createRole"
+
+	if options != nil {
+		if options.IfExist == "replace" {
+			action = "createOrReplaceRole"
+		} else if options.IfExist != "error" {
+			return types.Role{}, errors.New(fmt.Sprintf("Invalid value for the 'ifExist' option: '%s'", options.IfExist))
```

Comment:
This error does not make sense to me. The `options` object is directly passed to Kuzzle, which passes it down untouched to the InternalEngine. This one, in turn [only takes the `refresh` option into account](https://github.com/kuzzleio/kuzzle/blob/ce0ad380fa1e9c4ef12029468eb5d9716e913cfb/lib/services/internalEngine/index.js#L274-L274).
Actually, to me, there's no point in having this property being a String. IMHO, this should be called `replaceIfExists` and be a Boolean.